src/index.ts

```diff
@@ -234,7 +234,9 @@ class DB2Client extends knex.Client
             }
             catch (err : any)
             {
-                this.printError(err);
+                this.printError(err);                
+                console.error(err);
```

Comment:
Don't add a `console.log`, that's the point of `printError`.

Reply:
Should be fixed in latest commits.